opacus/accountants/utils.py

```diff
@@ -39,27 +38,31 @@ def get_noise_multiplier(
         raise NotImplementedError(
             "get_noise_multiplier is currently only supports RDP accountant"
         )
+    if (steps is None) == (epochs is None):
+        raise ValueError(
+            "get_noise_multiplier takes as input EITHER a number of steps or a number of epochs"
+        )
+    if steps is None:
+        steps = int(epochs / sample_rate)
 
     eps = float("inf")
-    sigma_min = DEFAULT_SIGMA_MIN_BOUND
-    sigma_max = DEFAULT_SIGMA_MAX_BOUND
     accountant = create_accountant(mechanism=accountant)
 
     while eps > target_epsilon:
-        sigma_max = 2 * sigma_max
-        accountant.steps = [(sigma_max, sample_rate, int(epochs / sample_rate))]
+        sigma_high = 2 * sigma_high
+        accountant.steps = [(sigma_high, sample_rate, steps)]
         eps = accountant.get_epsilon(delta=target_delta, **kwargs)
-        if sigma_max > MAX_SIGMA:
+        if sigma_high > sigma_max:
             raise ValueError("The privacy budget is too low.")
```

Comment:
Here we try to automatically initialize `sigma_high` for the binary search. This behaviour is not consistent with the docstring: I can set a specific bound but it can change it anyway. Also, having both `sigma_high` and `sigma_max` is counter-intuitive and it takes knowing the whole algorithm to understand what is the difference.

## Proposal

Leave only `sigma_low`, `sigma_high` in parameters. I would also rename to min/max to emphasize that the result will be bounded:

```python
def get_noise_multiplier(
    ...
    sigma_min: float = 0.01,
    sigma_max: Optional[float] = None,
)
```

If `sigma_max is None`, run this part to get the upper bound starting from `sigma_max+1`. 

Leave MAX_SIGMA a global constant, maybe make it higher.